src/module/remove.js

```diff
@@ -1,32 +1,51 @@
 import Data from './data.js';
 
-export default class Remove {
-  static #displayRemove = (data) => {
-    data.remove();
-  };
+export default class toDoDom {
+  static #datacheck = (done) => {
+    if (done) {
+      return 'done';
+    }
+    return '';
+  }
 
-  static #DataRemove = (data) => {
-    let datas = Data.dataGet();
-    const id = data.getAttribute('data-id');
-    datas = datas.filter((task) => task.id !== parseInt(id, 10));
+  static #addData = (data, container) => {
+    const listHtml = `
+    <li class="main-body-li" data-id="${data.id}">
+      <div class="width">
+        <span><input type="checkbox" name="list-check" id="list-check" ${toDoDom.#datacheck(data.done)}></span>
+        <span class="main-body-text"><input type="text" value="${data.text}" class="data-text ${toDoDom.#datacheck(data.done)}" contenteditable="true"</span>
+      </div>
+      <i class="fa-solid fa-square-minus remove-btn"></i>
```

Comment:
- [ ] I am unable to remove a task. When I click the button nothing happens. Kindly fix that.